proteuslib/tools/parameter_sweep.py

```diff
@@ -224,7 +224,7 @@ def parameter_sweep(m, sweep_params, outputs, results_file, optimize_fct,
 
     if debugging_data_dir is not None:
         # Create the local filename and data
-        fname = os.path.join(debugging_data_dir, 'local_results_%03d.csv' % (rank))
+        fname = os.path.join(debugging_data_dir, f'local_results_{rank}.csv')
```

Comment:
Is there an easy way to preserve the leading zeros using f-strings?  The sorting is a little bit nicer if they're there, and using the format modifiers gives us control over the output.